### ISSUE TITLE
Add visualization of electric fields

### ISSUE THREAD

**Issue by selenium2106** [niels747/2D-Weather-Sandbox#123]
Is there a way to add to the survey graph so that we can see how the electrical charges are distributed within the storm clouds? As well as the way to measure that electricity in kilovolts per meter with weather stations. Note: increasing the lightning generation rate is quite low, and it is also difficult to generate lightning storms.

**Comment by niels747** [comment]
No, because electric fields arent simulated. The lightning just spawns randomly, with the chance depending on the precipitation formation rate. I will soon add a lightning spwan chance setting. 
Maybe in the future I could try to simulate the complex nature of electic fields in thunderstorms.
